.storybook/helpers/files.data.ts

```diff
@@ -50,15 +72,30 @@ export const filesRoot: File[] = [
     ],
   },
   {
+    disabled: true,
+    selected: ClrSelectedState.UNSELECTED,
     name: 'package.json',
   },
   {
     name: 'tsconfig.json',
   },
 ];
 
-export function getFileTreeNodeMarkup(files: File[] = filesRoot) {
+export function getFileTreeNodeMarkup(files: File[] = filesRoot, useSelected?: boolean) {
   return files
-    .map(file => `<clr-tree-node>${file.name}${file.files ? getFileTreeNodeMarkup(file.files) : ''}</clr-tree-node>`)
+    .map(file => {
+      let selected = '';
+      if (useSelected !== undefined && file.selected !== undefined) {
```

Comment:
Why does this use the selected value if `useSelected` is false?

Reply:
For `clrSelected` on tree view we have 3 values `true`, `false` and `undefined` (not provided).
When is `undefined` the tree view is rendered as list. Basically no checkboxes.
When `clrSelected` have value `true` or `false` tree view have selection capabilities, through checkboxes.

Reply:
Okay, I would rename it to `clrSelected` to match the story.

Reply:
I might misunderstood the question. 
Let me check very quickly, but yeah renaming it to `clrSelected` makes much more sense.

Reply:
variable is renamed in commit 9e7f698

Reply:
Thanks!